node/npm/glide/package.json

```diff
@@ -58,5 +60,8 @@
             "build-ts/*"
         ],
         "root": true
+    },
+    "dependencies": {
```

Comment:
Please update readme and/or developer.md

Reply:
Will update README after it is released to npm. It wont be actually supported till the release so i prefer not to announce it.

Reply:
@avifenesh can you open an issue for that, so we won't forget to update the readme and developer files?

Reply:
Any follow up on this @avifenesh ? Should we add it to `node/package.json`? As it is used in `node/DEVELOPER.md` for `npm i`.

Reply:
Im not sure i understand. its in npm package.json, not part of node package. Its need for the index we use for the base package.

Reply:
So what is the recommended action plan for a developer to get rid of the squiggly lines underneath the import statement of `"detect-libc"` in `node/npm/glide/index.ts`? I tried to `cd` into `node/npm/glide` and do `npm i` but encoutered an `Invalid package name` error. I ended up executing `npm i detect-libc` which is not mentioned in `node/DEVELOPER.md` either.

Reply:
Usually we just ignore it. But if it bothers you go to package.json of npm, comment the "optionals", install, and remove comments.